### PR TITLE
Minor python packages updates

### PR DESCRIPTION
**Pull request by frjo**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,44 +1,46 @@
...
-django-countries==5.1
-Werkzeug==0.11.11
+# Development dependencies
 stellar==0.4.3
-django-tinymce4-lite==1.7.0
-uwsgidecorators==1.1.0
-django-hijack==2.1.9
-django-anymail==3.0
-celery==4.2.1
+Werkzeug==0.11.11
```
Comment (frjo) [inline requirements.txt]: Is this in use anywhere?

File: requirements.txt [hunk 2]
```diff
@@ -1,44 +1,46 @@
...
+django_select2==6.0.1
+djangorestframework==3.7.4
+django~=2.0.0
+gunicorn==19.9.0
 mailchimp3==3.0.4
-scout-apm==1.3.4
+Pillow==4.3.0
+psycopg2==2.7.3.1
+social_auth_app_django==2.1.0
+uwsgidecorators==1.1.0
```
Comment (frjo) [inline requirements.txt]: Is this in use anywhere?

**Comment by frjo** [comment]
@todd-dembrey are these three packages used anywhere?

* ConcurrentLogHandler
* uwsgidecorators
* Werkzeug

**Comment by todd-dembrey** [comment]
## Werkzeug
Used for `runserver_plus` from django_extensions: https://django-extensions.readthedocs.io/en/latest/runserver_plus.html

Dev requirement if you want to keep it.

## uwsgidecorartors
Can be removed

## ConcurrentLogHandler
Can be removed

**Comment by frjo** [comment]
Latest version of factory_boy, 2.11.1, generates many errors:

~~~~
TypeError: unroll_context() missing 1 required positional argument: ’self'
~~~~

Reverting to version 2.9.2 make it work again.

**Comment by frjo** [comment]
@todd-dembrey How do you handel this?

~~~~
The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
~~~~

**Comment by frjo** [comment]
Having issues updating to django-filter 2.x. The /submissions pages no longer works. Can't seem to find what changes are need to fix this. 

https://django-filter.readthedocs.io/en/master/guide/migration.html#migrating-to-2-0

**Comment by zerolab** [comment]
@frjo could the https://django-filter.readthedocs.io/en/master/guide/migration.html#filter-name-renamed-to-filter-field-name-792 change be the issue in [SubmissionFilter](https://github.com/OpenTechFund/opentech.fund/blob/master/opentech/apply/funds/tables.py#L130] ?

**Comment by frjo** [comment]
@todd-dembrey Do you see a problem with merging this? It's minor updates and I have run with these version locally for some time without issues.

I like to have the packages in some more logical order but the reorder makes this branch a hassel to rebase so would be good to get merged.

**Comment by frjo** [comment]
Good catch! Added it now.

**Comment by frjo** [comment]
Reverted the Django-tables2 2.x  update, too many issue for this PR.

**Pull request merged**